nav2_bt_navigator/behavior_trees/navigate_w_replanning_with_distance.xml

```diff
@@ -0,0 +1,14 @@
+<!--
```

Comment:
Since there's now a distinction between distance and time based replanning, it might be good to rename the navigate_w_replanning -> navigate_w_replanning_distance and this to navigate_w_replanning_time or something.

Reply:
That change should also be added to the migration guide.